ghga_service_commons/api/mock_router.py

```diff
@@ -98,21 +109,34 @@ class MockRouter:
 
     def __init__(
         self,
-        http_exception_handler: Optional[
-            Callable[[httpx.Request, HttpException], Any]
-        ] = None,
+        exception_handler: Optional[Callable[[httpx.Request, E], Any]] = None,
+        exceptions_to_catch: Optional[tuple[Type[Exception], ...]] = None,
+        handle_exc_subclasses: bool = False,
```

Comment:
```suggestion
        handle_exception_subclasses: bool = False,
```

Reply:
Changed in [91d166c](https://github.com/ghga-de/ghga-service-commons/pull/19/commits/91d166c7a9142872f908c00344217d34b5213580)